tests/suites/target_test.function

```diff
@@ -59,10 +59,43 @@ int verify_dependencies( uint8_t count, uint8_t * dep_p )
     return( DEPENDENCY_SUPPORTED );
 }
 
+/**
+ * \brief       Receives hex string on serial interface, and converts to a byte.
+ *
+ * \param none
+ *
+ * \return      unsigned int8
+ */
+uint8_t receive_byte()
+{
+    uint8_t byte;
+    uint8_t c;
+
+    c = greentea_getc();
+    if( c >= '0' && c <= '9' )
+        c -= '0';
+    else if( c >= 'a' && c <= 'f' )
+        c = ( c -'a' ) + 10;
+    else if( c >= 'A' && c <= 'F' )
+        c = ( c - 'A' ) + 10;
```

Comment:
Should we use `strtol()` instead of implementing ourselves? Is `strtol()` portable enough (e.g. win32, etc.)? Or should we use our `hexify()` and `unhexify()` functions?

Reply:
I'll check

Reply:
strtol is defined in Windows, I am not sure about other platforms.
However, `strtol` receive a nul terminated string, but `c` is only a character

Reply:
I used `unhexify()`